src/main/java/liqp/filters/Abs.java

```diff
@@ -10,9 +29,9 @@ public Object apply(Object value, Object... params) {
         }
 
         if (super.isNumber(value) || super.canBeDouble(value)) {
-            return Math.abs(super.asNumber(value).doubleValue());
+            return asFormattedNumber(new BigDecimal(super.asNumber(value).toString()).abs());
         }
 
-        return value;
+        return 0;
```

Comment:
Good to add the reference source, else I'd thought the `return 0;`  was incorrect :+1: